src/css/App.scss

```diff
@@ -48,6 +48,19 @@ ol, ul {
   }
 }
 
+.usa-input-error {
```

Comment:
Why do we use `usa-` namespace here? If it's something we write, shouldn't we use `coa-`?

Reply:
I JUST remembered i never responded to this. we're overwriting styles for a uswds component here, it's not our own component.

Reply:
cool, can we put these overrides maybe in the _uswds_variables file? Or another good place?